### PR TITLE
Feature contact validation

### PR DESCRIPTION
**Pull request by eduardosancho**
What was done in this PR:

-Remove 'required' property from form elements.
-Use javascript to validate the information inserted by the user.

- [x] Name field cannot be empty
- [x] Email field cannot be empty
- [x] Email field cannot have capital letters
- [x] Email field must have a valid email address. (following syntax abc@domain.com)
- [x] If a field has errors, a message is displayed near the submit button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -228,22 +228,22 @@ <h1> Get started</h1>
...
+                            <input type="text" id="email" name="user_email" placeholder="abc@email.com">
+                            <small></small>
                         </li>
                     </ul>
                     <label id="msg-label" for="msg">Message</label>
-                    <textarea id="msg" name="message" placeholder="Enter your message here..." maxlength="500"
-                        required></textarea>
-                    <button type="submit" id="submit-form">Start collaboration</button>
+                    <textarea id="msg" name="message" placeholder="Enter your message here..." maxlength="500"></textarea>
+                    <button type="submit" class="full" id="submit-form">Start collaboration</button>
```
Comment (oxenprogrammer) [inline index.html]: **Changes Required**
- [x] Please let us also try and validate `textarea` field. Apparently one can submit the form without any message.

Comment (oxenprogrammer) [reply index.html]: -  [x] Also your error message should be closer to the submit button has described in the requirements.

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @eduardosancho, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eduardosancho** [comment]
It seems linters are not working on GitHub right now.
I ran the checks locally and these were the results, everything passed fine.

image

**Review by Cathella — APPROVED** [review]
## 🟢 Status: Approved 🎉💪🏻

### Hi @team ✋🏻, 

You have done well implementing the requested changes 💪🏻, You form now submits and is well validated, Kudos to you 🎉🍾!
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**